### ISSUE TITLE
build-mono-docker.ps1 fails to run out the box

### ISSUE THREAD

**Issue by ChrisMaddock** [nunit/nunit-console#453]
I don't generally touch mono, but just tried out this script to debug https://github.com/nunit/nunit-console/pull/452. I followed the instructions in BUILDING.MD, which just say to install docker, and run `.\build-mono-docker.ps1 -t test`.

The script failed with the below message:

```
PS C:\Users\Chris\Documents\git\nunit-console> .\build-mono-docker.ps1 -t test
C:\ProgramData\chocolatey\lib\docker\tools\docker\docker.exe: error during connect: Post
 http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.33/containers/create: open //./pipe/docker_engine: 
The system cannot find the file specified. In the default daemon configuration on Windows, 
the docker client must be run elevated to connect. This error may also indicate that the 
docker daemon is not running.
See 'C:\ProgramData\chocolatey\lib\docker\tools\docker\docker.exe run --help'.
```

I've never touched Docker before, so have no idea what to do with that information! 😄 Anyone more familiar than I am have any guesses as to a fix we could put in?

**Comment by mikkelbu** [comment]
@ChrisMaddock Probably a stupid question :smile:, but did you run the docker client in elevated mode as the error message hinted?

**Comment by ChrisMaddock** [comment]
I ran the script in elevated mode, yep. I presume I don’t need to be running something else separately, alongside running this script?

**Comment by mikkelbu** [comment]
Running in an elevated PS shell I was possible to get all the way to the Tests task. There `TestNet20Engine` and `TestNet20Console` succeeded, but the NetStandard failed with the following errors. Perhaps we need to update @rprouse's image, I can see that he has been working on this lately https://github.com/rprouse/nunit-docker.

```
========================================
TestNetStandard16Engine
========================================
Executing task: TestNetStandard16Engine
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '1.1.2' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/share/dotnet/
  - Installing .NET Core prerequisites might help resolve this problem:
      https://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The .NET Core framework and SDK can be installed from:
      https://aka.ms/dotnet-download
  - The following versions are installed:
      2.1.6 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
An error occurred when executing task 'TestNetStandard16Engine'.

========================================
TestNetStandard20Engine
========================================
Executing task: TestNetStandard20Engine
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.0.6' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/share/dotnet/
  - Installing .NET Core prerequisites might help resolve this problem:
      https://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The .NET Core framework and SDK can be installed from:
      https://aka.ms/dotnet-download
  - The following versions are installed:
      2.1.6 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
An error occurred when executing task 'TestNetStandard20Engine'.
```

**Comment by rprouse** [comment]
@ChrisMaddock I got the same error message you did after a Docker upgrade. For some reason docker didn't start up after the update. A reboot fixed it.

@mikkelbu the error you reported slipped in with my updates to the docker image, sorry. I updated the image recently for the changes to the engine so that we had the newer .NET Core SDKs to build with. I didn't test with a few of the final changes though. It looks like I need to install .NET Core 1.1.2 and 2.0.6 into the image. I'll assign this to myself.
